_pkgdown.yaml

```diff
@@ -43,6 +43,7 @@ reference:
       - ProbitLogNormal
```

Comment:
need to add the new increments rule here

Reply:
done